<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>617</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icon.jpg</normaloff>:/icon.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="inputEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>971</width>
      <height>221</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="acceptRichText">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="outputEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>971</width>
      <height>201</height>
     </rect>
    </property>
    <property name="acceptRichText">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1032</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCtf_crypto_tools">
    <property name="title">
     <string>文字处理</string>
    </property>
    <addaction name="textReverse"/>
    <addaction name="toUpper"/>
    <addaction name="toLower"/>
    <addaction name="exchangeLowerUpper"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>编码解码</string>
    </property>
    <widget class="QMenu" name="menuBase16_32_64">
     <property name="title">
      <string>Base16/32/64</string>
     </property>
     <addaction name="base64Encode"/>
     <addaction name="base64Decode"/>
     <addaction name="separator"/>
     <addaction name="base32Encode"/>
     <addaction name="base32Decode"/>
     <addaction name="separator"/>
    </widget>
    <addaction name="menuBase16_32_64"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>加密解密</string>
    </property>
   </widget>
   <addaction name="menuCtf_crypto_tools"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="textReverse">
   <property name="text">
    <string>文字反向</string>
   </property>
  </action>
  <action name="toUpper">
   <property name="text">
    <string>大写</string>
   </property>
  </action>
  <action name="toLower">
   <property name="text">
    <string>小写</string>
   </property>
  </action>
  <action name="exchangeLowerUpper">
   <property name="text">
    <string>大小写互换</string>
   </property>
  </action>
  <action name="base64Encode">
   <property name="text">
    <string>Base64编码</string>
   </property>
  </action>
  <action name="base64Decode">
   <property name="text">
    <string>Base64解码</string>
   </property>
  </action>
  <action name="base32Encode">
   <property name="text">
    <string>Base32编码</string>
   </property>
  </action>
  <action name="base32Decode">
   <property name="text">
    <string>Base32解码</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
